scripts/refresh-after-snapshot.py

```
@@ -0,0 +1,644 @@
+        condition=lambda: subprocess.run(
+            ["oc", "apply", "-f", "-"],
+            input=pool_yaml, text=True, check=False, capture_output=True,
+            cwd=str(REPO_ROOT),
+        ).returncode == 0,
+    )
+    print("  MetalLB configured")
+
+
+def wait_keycloak_cert(config: RefreshConfig) -> None:
+    print("  Waiting for Keycloak TLS certificate...")
+    oc("wait", "--for=condition=Ready", f"certificate/keycloak-tls",
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)